### ISSUE TITLE
Add to F-Droid

### ISSUE THREAD

**Issue by maxkrieger** [maxkrieger/voiceliner#45]
Should probably add APK as release artifact. Also want to finish local transcription first.

https://f-droid.org/en/contribute/

**Comment by maxkrieger** [comment]
It sounds like gdrive and sentry integration might not be possible if we submit here? https://f-droid.org/en/docs/Inclusion_Policy/ In particular not sure if gdrive sign in required play services.

**Comment by lpstandardrp** [comment]
TL/DR:
I do not know how to get IzzyOnDroid's attention on Github; I have seen his help numerous times in Github Issues specifically, if I am not mistaken, in aiding to get an app on F-Droid.  

**Maybe someine can help(?!?) with this by lettimg him (@IzzyOnDroid ?) know I humbly request his help and attention with this specific issue** which I estimate could greatly benefit from his assistance! It no doubt would be ineffably appreciated. !
 
In looking up IzzyOnDroid's contact info on his website I stumbled upon this article which may be insightful, helpful and of value. Included is the Title*, URL*, and an excerpt from the article (in Markdown 'quote') which specifically highlights the use case of "getting the app on F-Droid" and how IzzyOnDroid was, and therefore might be able and willing to be, of assistance with Voiceliner:
**I used [Markdownr](https://www.f-droid.org/en/packages/com.sanzoghenzo.markdownr/)– an awesome and very useful Android app!– to convert the URL to Markdown:*
# [Metatrans Apps introduces its own F-Droid repository](https://android.izzysoft.de/articles/named/fdroid-metatrans?lang=en)

Clipped from: https://android.izzysoft.de/articles/named/fdroid-metatrans?lang=en
(To read in de/Deutsch/German](https://android.izzysoft.de/articles/named/fdroid-metatrans?lang=de)
> If you are developer, you can do everything by yourself, although it has learning curve and needs time. As I was not willing to wait and wanted to go for own store immediately, I requested help from Izzy. We agreed than in the beginning he will do everything (we achieve go live in just a few days that way) and than in next weeks and months he will handover to me tasks in order to educate me and I will be able to do the job by myself in future. Of course, I have appreciated his help and compensated for it.

(Longer message):
I'm so excited to see Voiceliner, I have been looking for an Android FOSS  "voice input" (i e. speech recognition for keyboards) which it seems Voiceliner currently doea not aupport (I may be incorrect, I just found this app and want to help however I can– hence this post); furthermore I have for a long time looking for a transcriber and... just wow. Wow. 

The idea and functionality and vision of this app I greatly sincerely hope continues tobunfold and I am excited beyond words at the prospect of making this an ongoing daily use. Finally, if Voiceliner maintains a FLOSS option (sans trackers) I have no doubt it has the prospect of being legendary, perhaps a desert island app... perhaps along the lines of Syncthing, Netguard, FairEmail, etc. (P.s. Netguard and FairEmail offers FLOSS and FOSS options, meaning donations are accepted in F-Droid / non-Google Play Store versions, and/or Google Play Store purchase, which both include benefits (obviously financial support to the developer(s)) not included in the Free/Standard version... I am all for free apps(!), *AND* at the same time I understand development takes considerable time and costs– I would rather an app survive and maintain development especially if donations *and* purchases make thatnpossible, especially if donations-only support limited the potential to keep an app developed, and alive for the benefit of all!  I appreciate what you are doing and again look forward to supporting however I am able and willing!

Lastly, the Voiceliner [Exodus Report](https://reports.exodus-privacy.eu.org/en/reports/search/chat.saga.voice_outliner/) - my praises and gratitudes again! 0 Trackers! **Keep up the great work (please :) )!!** 

That said... (see TL/DR above* :) )
